src/reducers/articleReducers/articlesReducers.js

```diff
@@ -43,11 +44,16 @@ const articlesReducerExtended = (state, action, allArticles) => {
 
 const articlesReducer = (state = articlesState, action) => {
 	const allArticles = state.results;
-
+	let { results = [] } = action.payload || {results: []};
```

Comment:
this is too complex, my friend @reiosantos. Find a way of making it easier to understand